### ISSUE TITLE
Why don't called "commands" instead of "plugins" directory

### ISSUE THREAD

**Issue by tansautn** [adityash4rma/flaregram#12]
For example: in the future. Some one wanna to contribute this project with his/her idea. "Implement a new feature just released by TG".

So. the current plugins directory (Assuming they do not have own repository, just pull then add commands).  It's can not update to remote repository anymore (local change conflict).

.....
Many, many cases i can imagine.

Also. call "plugins" does not cleary in meaning name. While plugin just mean "attach/detachable" and it's also "does not required, the project can work with out plugin".

**Comment by adityash4rma** [comment]
Hey @tansautn,

> For example: in the future. Some one wanna to contribute this project with his/her idea. "Implement a new feature just released by TG".
> So. the current plugins directory (Assuming they do not have own repository, just pull then add commands). It's can not update to remote repository anymore (local change conflict).

what i could understand from this is that you are confused that how will a contributor add the new features implemented my telegram on its Bot API, so, to begin with, the API Methods provided by telegram is in this directory,

> ./flaregram/tree/main/src/flaregram/api

![image](https://github.com/adityash4rma/flaregram/assets/102592458/d923825b-091b-4740-b159-6103214a9aab)
Above is the list of API Methods I wrote to interact with telegram API.
So, if you want to contribute, you can check the methods and in the `./api/` directory! 😄 

---

>.....
>Many, many cases i can imagine.
>Also. call "plugins" does not cleary in meaning name. While plugin just mean "attach/detachable" and it's also "does not required, the project can work with out plugin".

Moving on, I named the `plugins` directory, as `plugins` and not `commands` it's because, IT IS `attachable/detachable` by user and also its simply not only just filled up with `commands`, each plugin can also do more stuff in the backend, so calling it only, `commands` wouldn't suit it (my personal thought).

So, to summarize it up! 

`plugins directory` ≠ `api directory`
Which means `plugins` contains user defined commands
and `api` directory contains the API methods for interacting with telegram api

Hope it helps! You can ask anything else if you like! 😉
